.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
```

Comment:
Thinking here that we could name this workflow in the same way that we did for Scribe-Data actually, [`pr_ci.yaml`](https://github.com/scribe-org/Scribe-Data/blob/a4bad64dee5e8605e884b10b73fbf2bb963def0a/.github/workflows/pr_ci.yaml#L1)
We could name this `pr_ci` as well and the file `pr_ci.yaml`

Reply:
okay